src/actions.hpp

```diff
@@ -92,7 +92,7 @@ namespace Action {
         virtual Task* clone_() const =0;
         
         //! copy binary_ from command-line params to task
-        bool binary_ ;
+        bool binary_ = false;
```

Comment:
This cannot be done with c++98/03, the travis-ci builds are complaining about it appropriately:

```bash
/home/travis/build/Exiv2/exiv2/src/actions.hpp:95:24: error: non-static data member initializers only available with -std=c++11 or -std=gnu++11 [-Werror]
         bool binary_ = false;
```
Therefore, you will need to initialise this variable in the constructor initialisation list